### ISSUE TITLE
Consider underlining links in paragraphs

### ISSUE THREAD

**Issue by henrikhelmers** [poole/poole#62]
Accessibility would be better if text is underlined in running text. There is quite a bit of contrast between the black text and the blue color, but without color vision or in poor light it looks more disabled than clickable.

Something like this should do, and still not interfere with the more graphical elements:

```
p a {
  text-decoration: underline;
}
```
